packages/web-pkg/src/components/CreateLinkModal.vue

```diff
@@ -69,14 +69,47 @@
       @click="$emit('cancel')"
       >{{ $gettext('Cancel') }}
     </oc-button>
-    <oc-button
-      class="link-modal-confirm oc-modal-body-actions-confirm oc-ml-s"
-      appearance="filled"
-      variation="primary"
-      :disabled="confirmButtonDisabled"
-      @click="$emit('confirm')"
-      >{{ confirmButtonText }}
-    </oc-button>
+    <div
+      class="link-modal-confirm-button-group oc-ml-s"
+      :class="{ 'oc-button-group': password.value }"
+    >
+      <oc-button
+        class="link-modal-confirm oc-modal-body-actions-confirm"
+        appearance="filled"
+        variation="primary"
+        :disabled="confirmButtonDisabled"
+        @click="onConfirm"
```

Comment:
Please keep using `$emit('confirm')` to make sure the confirm method of the modal wrapper gets called. It handles things like loading service and closing the modal afterwards. You can simply pass an argument for copying the password if needed.

Reply:
This ain't working for me :/ I literally cant pass the argument? Could you contribute ?

Reply:
Oh hm, let me try

Reply:
There was something missing in the modal wrapper component 😬 Now it should work!

Reply:
Ahhh thx big times =) !